app/src/main/java/com/example/inoutshoppers/item/AddItem.kt

```diff
@@ -114,16 +114,20 @@ class AddItem : Fragment() {
                         Log.i(TAG, "Current location: $location")
                         var item = binding.itemNameEdit.text.toString()
                         // Add to database
-                        dao.addItem(selectedStore!!, item, location!!,
-                            {
-                                Toast.makeText(requireContext(), "$item added to ${selectedStore!!.name}", Toast.LENGTH_LONG).show()
-                                view.findNavController()
-                                    .navigateUp()
-                            },
-                            {
-                                Toast.makeText(requireContext(), "Unable to add item. Please try again.", Toast.LENGTH_LONG).show()
-                            })
-
+                        if (location == null) {
```

Comment:
I tried running the fix on the emulator. The app isn't crashing anymore. Thanks!